src/nxdoc/nuxeo-server/administration/monitoring-and-maintenance/hotfixes-installation-notes-for-nuxeo-platform-lts-2019.md

```diff
@@ -656,6 +656,8 @@ SELECT nx_rebuild_read_acls();
 
 ### StreamWorkManager Configuration
 
+Prerequisite: To apply this configuration the `nuxeo.conf` should contain `nuxeo.stream.work.enabled=true`.
```

Comment:
Isn't it implicite that nuxeo.stream.work.enabled=true is needed to use StreamWorkManager.

This line is correct. However if we add it, shouldn't we add the prerequisite for all notes in the page, for example the notes about S3?

Reply:
I understand and it's fair, I have spend some time to understand the property `nuxeo.stream.work.enabled` existed and both were dependent. I think it should be mentioned.

What I noticed also is that this feature is a temporary to circumvent with kafka and chronicle queue limitation and help nuxeo get around with big records until they are properly refactored to be smaller. 
Maybe we should mention that this configuration should be used wit care. Some limitations are listed below.

See: https://jira.nuxeo.com/browse/NXP-26691

> Again this is a workaround until works are fixed because having the record value outside of the record have consequences:
> * the log is not anymore the single source of trust
> * failover based on Log replication will depend on an additional storage replication
> * retention of the additional storage must match the Log retention
> * constant throughput will be impaired by mixing very different record sizes and access